test/integration/apis/issue.test.ts

```diff
@@ -51,32 +62,13 @@ describe("issue", () => {
             assert.isRejected(issueAPI.request(amount));
         });
 
-        it("should page listed requests", async () => {
-            issueAPI.setAccount(alice);
-            const bobVaultId = api.createType("AccountId", bob.address);
-            const sentRequests = 3;
-            for (let i = 0; i < sentRequests; i++) {
-                const amount = api.createType("Balance", i);
-                await issueAPI.request(amount, bobVaultId);
-            }
-
-            const listingsPerPage = 2;
-            let requestCount = 0;
-            for await (const page of issueAPI.getPagedIterator(listingsPerPage)) {
-                requestCount += page.length;
-                assert.isTrue(page.length <= listingsPerPage);
-            }
-            assert.equal(requestCount, sentRequests);
-        });
-
-        it("should retrieve hash from request", async () => {
+        it("should request issue", async () => {
             keyring = new Keyring({ type: "sr25519" });
-            bob = keyring.addFromUri("//Bob");
             alice = keyring.addFromUri("//Alice");
             issueAPI.setAccount(alice);
-            const bobVaultId = api.createType("AccountId", bob.address);
-            const amount = api.createType("Balance", 1);
-            const requestResult = await issueAPI.request(amount, bobVaultId);
+            const amount = api.createType("Balance", 100000);
+            const requestResult = await issueAPI.request(amount);
+            console.log("finalized request issue");
             assert.isTrue(requestResult.hash.length > 0);
```

Comment:
Better would be to check the request issue event for the actual input data. Maybe best would be to expose the helper functions inside the issue api to check for matching events here?

Reply:
I'll add a check in the issue API that throws an error if events are not ok, so this will automatically invalidate any failing tests